api-docs/client-interactions.md

```diff
@@ -163,14 +140,12 @@ POST /api/clients/:id/interactions
 ```json
 {
   "id": 1,
-  "title": "Nutrition Program - SNAP",
   "serviceId": 7,
   "interactionType": "walkIn",
   "description": "Application successfully filled out. Decision pending.",
   "dateOfInteraction": "2019-05-06",
   "duration": "1:00:00",
   "location": "CUOffice",
-  "canModify": true,
   "isDeleted": false,
```

Comment:
Does the isDeleted need to be removed here also?

Reply:
No, since this is a response object, not a request object.